Java/sort/CombSort_implementation_in_java.java

```diff
@@ -0,0 +1,77 @@
+/*
+    Comb Sort is a relatively simple sorting algorithm. It is an improvement over Bubble sort
+    by using gap of size more than 1. The gap starts with a large value and starts shrinking by
+    1.3 in every iteration until it reaches the value 1.
+    You can check out more about this Sorting Algorithm here:- https://en.wikipedia.org/wiki/Comb_sort
+*/
+
+import java.util.Scanner;
+
+public class CombSort_implementation_in_java {
```

Comment:
Kindly Renamed `CombSort_implementation_in_java` to `CombSort_implementation`

Reply:
@ankitaggarwal23 Done. Is it fine?

Reply:
One more left. Just Fix that and We are good to merge this PR. @Devansu-Yadav

Reply:
@ankitaggarwal23 Done!

Reply:
@ankitaggarwal23 Done!